.travis.yml

```diff
@@ -29,7 +31,14 @@ matrix:
             - scripts/ci-localerefresh
             - bash <(curl -s https://codecov.io/bash)
         - language: node_js
+          name: "Node (lint)"
           node_js: 10
           cache: npm
           script:
             - scripts/ci-jslint
+        - language: node_js
+          name: "Node (test)"
+          node_js: 10
+          cache: npm
+          script:
+            - scripts/ci-jest
```

Comment:
Move this line to be between line 30 and 31. And remove this whole matrix node. It's not actually "python" (although it says that on line 17. It's just Python as the foundation before starting docker-compose.

Reply:
Aren't we going to loose the ability to run those in parallel ?

Reply:
We are, but the "python block" already does all the docker-compose stuff. Remember, docker-compose isn't just python stuff. For example, you need to run the webpack build so that Django has the file it needs. 

We *could* do the docker-compose stuff too, in parallel, in this block but it'd probably take longer in total.